app/services/swiss_implementation.rb

```diff
@@ -8,9 +8,10 @@ def self.pair(players, options = {}, &block)
   end
 
   class Player
-    attr_accessor :delta, :exclude, :label
+    attr_accessor :id, :delta, :exclude, :label
```

Comment:
It looks like this model is only used in tests? Can we update those to use PlainPlayer instead and remove this?

Reply:
let's do that in a follow up PR.  Again, this review is dragging on and that won't affect functionality.